pod/meeting/views.py

```diff
@@ -485,6 +472,87 @@ def get_meeting_info(request, meeting_id):
         return JsonResponse({"info": info, "msg": msg}, safe=False)
 
 
+@login_required(redirect_field_name="referrer")
+def recordings(request, meeting_id):
+    meeting = get_object_or_404(
+        Meeting, meeting_id=meeting_id, site=get_current_site(request)
+    )
+
+    if request.user != meeting.owner and not (
+        request.user.is_superuser or request.user.has_perm("meeting.delete_meeting")
+    ):
+        messages.add_message(
+            request, messages.ERROR, _("You cannot delete this meeting.")
+        )
+        raise PermissionDenied
+
+    meeting_recordings = meeting.get_recordings()
+    recordings = []
+    if type(meeting_recordings.get("recordings")) is dict:
+        for record in meeting_recordings.get("recordings"):
```

Comment:
si pas besoin de la clé, on peut faire directement ça non ?
`for data in meeting_recordings.get("recordings").values():`

Reply:
je vais tester, bien vu !